javagen/src/main/java/com/azure/autorest/template/ProtocolSampleTemplate.java

```diff
@@ -103,22 +123,29 @@ public void write(ProtocolExample protocolExample, JavaFile javaFile) {
                     credentialExpr = "";
                 }
 
+                // client
                 String clientInit = "%s client = new %s()" +
                         ".%s(System.getenv(\"%s\"))" +
                         "%s" +
```

Comment:
Remaining issue on the filling of parameters that set as client property (like instanceId in device update).